typescript.ts

```diff
@@ -0,0 +1,5 @@
+export type alertType = {
+     msg: string,
+     alertType: string,
+     id: string
+}
```

Comment:
Bad naming for file. As I understand here should be some TS configs and common interfaces... If application will be big then this file will be very big too... No navigation in this case

Reply:
fixed